spec/models/user_spec.rb

```diff
@@ -1,24 +1,32 @@
-require 'rails_helper'
+require "rails_helper"
 
 RSpec.describe User, type: :model do
   before :each do
     @user = FactoryGirl.create(:user)
   end
 
-  it 'should be valid without a full name' do
+  it "should be valid without a full name" do
     expect(@user).to be_valid
   end
 
-  it 'should be invalid without a full name' do
+  it "should be invalid without a full name" do
     @user.full_name = nil
     expect(@user).not_to be_valid
   end
 
-  it 'should have many orders' do
+  it "should have many orders" do
     expect(@user).to respond_to(:orders)
   end
 
-  it 'should have many items' do
+  it "should have many items" do
     expect(@user).to respond_to(:items)
   end
+
+  it "it displays placed orders for user" do
+    item = FactoryGirl.create(:item)
+    order = FactoryGirl.create(:order, user_id: @user.id)
+    order_item = FactoryGirl.create(:order_item, item_id: item.id, order_id: order.id)
```

Comment:
Useless assignment to variable - `order_item`. Did you mean `order`?<br>Line is too long. [86/80]